### ISSUE TITLE
replace all intra doc links with internal links - rst format

### ISSUE THREAD

**Issue by phillxnet** [rockstor/rockstor-doc#283]
Further to @FroggyFlox's recent audit of our links, we have some outstanding URL's within the docs that are external links to our production 'self'. This is inconsistent and circumvents embedded spinx link checks/creation and leads to redundant arbitrary checks of the published docs. Given sphinx auto-generates internal links to be relative we can thus completely avoid these unnecessarily hard coded links.

**Comment by FroggyFlox** [comment]
After a quick search for the `rockstor.com/docs` string in the repo, I could only see 2 instances:
- `home_nas_guide.rst`
- `afp/apple_filing_protocol.rst`
The latter is actually part of a `.. raw:: html` block, so the full html link is actually needed and we can't use internal ref links.

Let me know if you spotted other ones.

**Comment by phillxnet** [comment]
@FroggyFlox Thanks for taking a look at this. I'd not realised there was now only one remaining. I created this issue after noticing that I'd missed at least one of these in my review of your doc links work pr:

Fix broken links (#278) #282

And rather than have another round of review on the same pr where your focus was no this element of links anyway I just bashed out this issue to catch what I'd missed in the review and get your more important http -> https & broken links improvements in. These in turn related to the host server moving to https which was also addressed in my own prioritised changes re http -> https  for the website, as they share this common hosting move from http -> https:

Address mixed http https content https://github.com/rockstor/rockstor-website/pull/13

So this internal link issue was simply split out to add no more delay / back-and-forth on my already delayed review of you doc pr.

Bit tricky having these disparate but do-dependant repos: website & doc, but given they each have very different processors I think it's best we keep them separated. But the recent http -> https hosting rather forced our hand on getting both repos content in line there after.
